static/static/berlin.json

```diff
@@ -0,0 +1,94 @@
+{
+  "api": "0.5.2",
+  "contact": {
+    "email": "info@berlin.freifunk.net",
+    "ml": "berlin@berlin.freifunk.net",
+    "twitter": "@freifunk_berlin",
+    "matrix": "https://matrix.to/#/#berlin.freifunk.net:matrix.org"
+  },
+  "location": {
+    "address": {
+      "name": "Haus der Materialisierung",
+      "street": "Otto-Braun-Str. 70-72",
+      "zipcode": "10178"
+    },
+    "city": "Berlin",
+    "country": "DE",
+    "geoCode": {
+      "lat": 52.52315,
+      "lon": 13.41957
+    }
+  },
+  "name": "Freifunk Berlin",
+  "nodeMaps": [
+    {
+      "interval": "daily",
+      "mapType": "geographical",
+      "technicalType": "openwifimap",
+      "url": "https://openwifimap.net"
+    },
+    {
+      "interval": "daily",
+      "mapType": "geographical",
+      "technicalType": "hopglass",
+      "url": "https://hopglass.berlin.freifunk.net"
+    }
+  ],
+  "socialprojects": {
+    "number": 38,
+    "website": "https://wiki.freifunk.net/Berlin:Refugees"
+  },
+  "state": {
+    "focus": [
+      "infrastructure/backbone",
+      "Public Free Wifi",
+      "Free internet access"
+    ],
+        "lastchange":"2023-12-14T19:01:04+01:00",
```

Comment:
This is dynamic data and should not be hardcoded like that. In the old setup this was:
```javascript
    "lastchange": "FILLED_IN_BY_MAKEFILE",
    "nodes": "FILLED_IN_BY_MAKEFILE"
```
We should also find a solution to update this data. Daily might be a good interval.

Reply:
We should add this to the workflows and make it scheduled, so that this gets rebuild every day.

Reply:
I moved the commands from the old makefile into the workflow. This is working, but I think a solution where hugo renders that file and gets some vars from the environment might be better.

Reply:
I think we should just make this an issue to keep track of it and fix it later. In my opinion we can merge without this.

Reply:
I removed the wip solution from the workflows. Implementing the better (hugo render based) solution is now an issue at #86.